stan/math/prim/scal/fun/lgamma_stirling.hpp

```diff
@@ -10,6 +10,7 @@ namespace math {
 
 template <typename T>
 T lgamma_stirling(const T x) {
+  using std::log;
   return 0.5 * log(2 * stan::math::pi()) + (x - 0.5) * log(x) - x;
```

Comment:
Note that we have `LOG_TWO_PI` defined in `stan/math/prim/scal/fun/constants.hpp`.

Reply:
Thanks for the input. Sorry, the code is now a bit of a mess, #1592 sent me on a wild tangent and the current state is more of a prototype than polished code.

Reply:
No problem, I was not trying to review, just browsed the code and saw that! Ah, perhaps there'll be a HALF_LOG_TWO_PI constant soon. :)